src/main/java/com/jalasoft/webservice/model/ImageConvert.java

```diff
@@ -61,13 +52,13 @@ public String ImageConvert(String sourcePath, int dpi, String destinationPath, S
 
             //Instantiating the PDFRenderer class
             PDFRenderer renderer = new PDFRenderer(document);
-            for (int page = 0; page < document.getNumberOfPages(); ++page){
+            for (int page = 0; page < document.getNumberOfPages(); ++page) {
```

Comment:
Review the document.getNumberOfPages() and put it as a variable before in order to achieve a better performance.

Reply:
Can you review the new commit with the suggestion changes?